src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_pagedattn_inputs.cpp

```diff
@@ -0,0 +1,137 @@
+                                    const size_t group_size,
+                                    const bool bychannel) {
+            ov::Dimension::value_type block_size = 32;
+            ov::Dimension::value_type _head_nums = head_nums;
+            ov::Dimension::value_type _head_size = head_size;
+            ov::Dimension::value_type _group_size = group_size;
+            _group_size = _group_size ? _group_size : _head_size;
+            if (!bychannel) {
+                if (_head_size % _group_size != 0) {
+                    OPENVINO_THROW("cache head_size ", head_size, "cannot be divided by group_size ", group_size);
+                }
+            }
+            size_t group_num = _head_size / _group_size;
+            if (precision == ov::element::u8) {
+                if (bychannel) {
+                    block_size += 2 * sizeof(float);
+                } else {
+                    _head_size += sizeof(float) * 2 * group_num;
+                }
+            } else if (precision == ov::element::u4) {
+                _head_size += sizeof(float) * 2 * group_num * 2;
+            }
+            return ov::PartialShape{-1, _head_nums, block_size, _head_size};
+        };
+        auto key_cache_precision = format_cache_precision(config.keyCachePrecision, config.inferencePrecision);
+        auto value_cache_precision = format_cache_precision(config.valueCachePrecision, config.inferencePrecision);
+        key_cache->set_element_type(key_cache_precision);
+        value_cache->set_element_type(value_cache_precision);
+        if (!pa_op->get_rt_info().count("num_k_heads") || !pa_op->get_rt_info().count("k_head_size") ||
+            !pa_op->get_rt_info().count("num_v_heads") || !pa_op->get_rt_info().count("num_v_heads")) {
+            OPENVINO_THROW("PagedAttn ",
+                           pa_op->get_friendly_name(),
+                           " doesn't have rtinfo for num_k_heads/k_head_size/num_v_heads/num_v_heads");
+        }
+        const auto key_cache_shape = init_cache_shape(pa_op->get_rt_info()["num_k_heads"].as<size_t>(),
+                                                        pa_op->get_rt_info()["k_head_size"].as<size_t>(),
+                                                        key_cache_precision,
+                                                        config.keyCacheGroupSize,
+                                                        false);
+        const auto value_cache_shape = init_cache_shape(pa_op->get_rt_info()["num_v_heads"].as<size_t>(),
+                                                      pa_op->get_rt_info()["v_head_size"].as<size_t>(),
+                                                      value_cache_precision,
+                                                      config.valueCacheGroupSize,
+                                                      false);
+
+        key_cache->set_partial_shape(key_cache_shape);
+        value_cache->set_partial_shape(value_cache_shape);
```

Comment:
will this information be passed to `CompiledModel::inputs()` ?

As I see, CPU plugin does override there methods:
https://github.com/openvinotoolkit/openvino/blob/e737014105e300628201daa13dde9127b6187caf/src/inference/dev_api/openvino/runtime/icompiled_model.hpp#L79-L91
and original `ov::Model`'s precisions / shapes will be used.

Reply:
I think the info is passed to compiled_model after compile successfully.  The test case in the PR proves that the info is passed correctly. 
```
for (const auto& input : compiledModel.inputs()) {
        for (auto& name : input.get_names()) {
            auto cachePrec = input.get_element_type();
           ....
}
```
Meanwhile, in src/plugins/intel_cpu/src/compiled_model.h, the `inputs()` and `output()s` method are not overrided